### ISSUE TITLE
setState() called after unmount

### ISSUE THREAD

**Issue by timscott** [olofd/react-native-device-log#7]
[This line](https://github.com/olofd/react-native-device-log/blob/master/debug-view.js#L25) is being called after unmount.  It's probably harmless, but might be something to clean up.

```
Warning: setState(...): Can only update a mounted or mounting component. This 
usually means you called setState() on an unmounted component. This is a no-op.
Please check the code for the DebugView component.
```

**Comment by olofd** [comment]
Ah. Thanks. Will fix that.

**Comment by olofd** [comment]
@timscott Sry for the wait. But it's fixed and released under v0.0.17 on npm.
